src/writeData/constants/contentTelegrafPlugins.ts

```diff
@@ -1077,12 +1075,6 @@ export const WRITE_DATA_TELEGRAF_PLUGINS: TelegrafPluginAssets[] = [
     markdown: monitMarkdown,
     image: monitLogo,
   },
-  {
```

Comment:
I believe we should still display this plugin. [This](https://www.influxdata.com/integration/mqtt-telegraf-consumer/) looks to be for cases where a customer is using telegraf to collect MQTT data and output it to an InfluxDB instance (including Cloud2) as line protocol.